_devconfig/ssh-all-2.md

```diff
@@ -0,0 +1,209 @@
+
+     > _This automatically fills in the **Cert** and **SSH keystring** textboxes._
+
+  2. Copy and paste the **SSH keystring** **_value_** (i.e., SSH key) into **Microsoft Notepad** and save it.  
+  3. Provide your SSH key to the Jump server administrator and ask that it be added to your Jump server account.
+
+     > _**Note:**  Once the Jump server account has been set up with your SSH key, you can use your PIV/CAC to log in. For other Jump servers, submit a service ticket to the administrator and include the IP address of the Jump server you are using, your account name, and your PIV/CAC's SSH key._
+
+#### Verify your PuTTY login and proceed with SSH
+
+  1. Run **PuTTY** and select the **Saved Session**.
+  2. Click on **Load** and then on **Open**.
+  3. Enter your **remote UNIX/Linux account name**.  
+  4. At the prompt, enter your **PIV card PIN** and click on **OK** to log into the remote server.
+  5. Once logged in, run the command: **ssh-add –l** to display the SSH key.  
+
+     > _For each server you "jump" to, use **ssh-add –l** to display the SSH key. Once you see the key, you may **ssh** to any other hosts in the environment._
+
+## Use PIV/CAC for SSH from a Linux Computer
+
+### Hardware and software requirements
+
+  * A PIV/CAC
+  * A smartcard reader
+  * A Linux computer or workstation that is configured to use a PIV/CAC card for login. (For additional information, go to [**configure opensc**](https://github.com/OpenSC/OpenSC/wiki/Download-latest-OpenSC-stable-release).)
+
+### Procedures
+
+These steps will help you to:
+
+  * Authenticate your PIV/CAC
+  * Enable you to use SSH for remote access
+
+#### Obtain and save public key from PIV/CAC
+
+  1. Insert your PIV/CAC into your computer's smartcard reader.
+  2. To save your **public SSH key** to a file, enter:
+
+        ```
+			ssh-keygen -D /usr/lib64/opensc-pkcs11.so > mykey.pub
+        ```  
+  3. Submit the file with your public SSH key to the Jump server administrator.
+
+#### Log in via SSH
+
+  1. Insert your **PIV/CAC** into your computer's smartcard reader.
+  2. To log into the remote server, enter:
+
+        ```
+			ssh -I /usr/lib64/opensc-pkcs11.so <remote-host>
+        ```    
+
+  3. At the PIV card password prompt, enter your **PIN**.
+
+     > _The **remote-host shell prompt** appears._
+
+{% include alert-warning.html heading = "The card reader may flash. **Do not** remove the PIV/CAC until the login process has been completed." %}&nbsp; 
+
+## Use PIV/CAC for SSH from a Mac OS X Computer
+
+### Hardware and software requirements
+
+  * A PIV/CAC
+  * A smartcard reader
+  * A Mac OS X computer configured for PIV/CAC login. For additional information, go to [**configure opensc**](https://github.com/OpenSC/OpenSC/wiki/Download-latest-OpenSC-stable-release).
```

Comment:
what version? need to update the drivers page but is this for Sierra? Or between Lion and Sierra?